frontend/util.lua

```diff
@@ -982,6 +982,15 @@ function util.getScriptType(file)
     end
 end
 
+--- A wrapper to os.execute
+function util.execute(...)
```

Comment:
We already have a util.execute() in base/ffi/util.lua - so even if this one's argument may be different, this could become confusing.
Also see FileManager:moveFile() in frontend/apps/filemanager/filemanager.lua.
We also have a util.shell_escape(args) used in readerdictionary.lua, which might be better at handling the quoting you do below (or not, dunno :).
Anyway, using the shell like you do here (and like we do elsewhere, because nobody cleaned that up :) is a bit odd and ugly, when Lua has a os.rename().